doc/filters/format_currency.rst

```diff
@@ -20,25 +20,85 @@ You can pass attributes to tweak the output:
 
 The list of supported options:
 
-* ``grouping_used``;
-* ``decimal_always_shown``;
-* ``max_integer_digit``;
-* ``min_integer_digit``;
-* ``integer_digit``;
-* ``max_fraction_digit``;
-* ``min_fraction_digit``;
-* ``fraction_digit``;
-* ``multiplier``;
-* ``grouping_size``;
-* ``rounding_mode``;
-* ``rounding_increment``;
-* ``format_width``;
-* ``padding_position``;
-* ``secondary_grouping_size``;
-* ``significant_digits_used``;
-* ``min_significant_digits_used``;
-* ``max_significant_digits_used``;
-* ``lenient_parse``.
+* ``grouping_used``: Specifies whether to use grouping separator for thousands.
+
+.. code-block:: twig
```

Comment:
I'm quite sure the way this markup is written means that we don't have 1 list of options anymore but one list between each code blocks. The code blocks should be inside each list item.